### PR TITLE
Updated oke run to use OL8.x image

### PR DESCRIPTION
**Pull request by marinakog**
added solution for Istio in OL8.x, updated oke run to use OL8.x image
https://build.weblogick8s.org:8443/job/wko-oke-nightly-parallel/527

https://build.weblogick8s.org:8443/job/wko-kind-nightly-parallel/1256

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/InitializationTasks.java [hunk 1]
```diff
@@ -335,7 +336,7 @@ public void close() {
     if (SKIP_CLEANUP) {
       logger.info("Skipping RESULTS_ROOT clean up after test execution");
     } else {
-      if (!OKD) {
+      if (!OKD && !OKE_CLUSTER) {
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/InitializationTasks.java:339]: Why are we skipping istio un-nstallation on OKE ?

Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/InitializationTasks.java:339]: it causes hanging the shell script after adding that option --set components.cni.enabled=true, OKD also skip uninstall

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/IstioUtils.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public static void installIstio() {
     assertTrue(() -> Command.withParams(
         defaultCommandParams()
             .command(command)
-            .redirect(false))
+            .redirect(true))
         .execute());
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/IstioUtils.java:77]: update copyright year

File: integration-tests/src/test/resources/bash-scripts/uninstall-istio.sh [hunk 3]
```diff
@@ -26,6 +26,7 @@ echo "Uninstalling Istio version [${version}] from location [${istiodir}]"
   bin/istioctl x uninstall --purge -y
```
Comment (sankarpn) [inline integration-tests/src/test/resources/bash-scripts/uninstall-istio.sh:26]: update copyright year

**Review by sankarpn — CHANGES_REQUESTED** [review]
minor comments

File: Jenkinsfile.oke [hunk 4]
```diff
@@ -64,9 +64,8 @@ pipeline {
         )
         string(name: 'IMAGE_ID',
                        description: 'oci image id for node pool, find image OCID for your region from https://docs.oracle.com/iaas/images/',
-                       defaultValue: 'ocid1.image.oc1.phx.aaaaaaaancucg25pze6m52agxwxmmfszvsdnhp3kvim53pts2tw6mik4xtha'
-                       //defaultValue8.0: 'ocid1.image.oc1.phx.aaaaaaaancq36l7q4fqwznqhnp626wuekkq7twk2em5ysyd65bd4xho72tmq'
-            
+                       //defaultValue7.6: 'ocid1.image.oc1.phx.aaaaaaaancucg25pze6m52agxwxmmfszvsdnhp3kvim53pts2tw6mik4xtha'
```
Comment (anpanigr) [inline Jenkinsfile.oke:67]: Which one is default now  ? OL 7.6 or OL 8.0 
We should make OL 8.0  as default

Comment (marinakog) [reply Jenkinsfile.oke:67]: yes it is default 8.0, 7.6 is commented

Comment (marinakog) [reply Jenkinsfile.oke:67]: ocid1.image.oc1.phx.aaaaaaaancq36l7q4fqwznqhnp626wuekkq7twk2em5ysyd65bd4xho72tmq is default now

**Review by anpanigr — APPROVED** [review]

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**